webapp/first_snap/content/java/package.yaml

```diff
@@ -0,0 +1,69 @@
+name: freeplane-{name}
+download: git clone https://github.com/snapcraft-docs/freeplane
+createDir: |
+  cd freeplane
+  $EDITOR snapcraft.yaml
+metadata: |
+  name: freeplane
+  version: git
+  summary: Application for Mind Mapping, Knowledge and Project Management
+  description: |
+    Freeplane is a free and open source software application that supports thinking, sharing information and getting things done at work, in school and at home. The core of the software is tools for mind mapping (also known as concept mapping or information mapping) and using mapped information.

```

Comment:
This should be wrapped, but it's not:
<img width="668" alt="screenshot 2019-02-11 at 11 48 23" src="https://user-images.githubusercontent.com/1523179/52561328-f5c25500-2df2-11e9-9f47-37c65bd2a8de.png">

Reply:
I've uploaded a new revision. Will fix the wrapping.

Reply:
That commit is missing. It's still not showing correctly. Make sure you `git add`ed the files to stage, committed, and pushed.

Reply:
Heads up this is still not fixed in the most recent set of commits.